vectortile/src/main/scala/geotrellis/vectortile/Layer.scala

```diff
@@ -127,16 +127,16 @@ import geotrellis.util.annotations.experimental
   private def unfeature(
     keys: Map[String, Int],
     values: Map[Value, Int],
-    geomType: vt.Tile.GeomType,
+    geomType: vector_tile.Tile.GeomType,
     cmds: Seq[Command],
     data: Map[String, Value]
-  ): vt.Tile.Feature = {
+  ): vector_tile.Tile.Feature = {
     val tags = data.toSeq.foldRight(List.empty[Int]) { case (pair, acc) =>
       /* These `Option.get` _should_ never fail */
       keys.get(pair._1).get :: values.get(pair._2).get :: acc
     }
 
-    vt.Tile.Feature(None, tags, Some(geomType), Command.uncommands(cmds))
+    vector_tile.Tile.Feature(None, tags, Some(geomType), Command.uncommands(cmds))
```

Comment:
This line assumes that features never have an associated `id`.  We need to have a means to transmit this information, but it may require something like the work in #2892 plus an id-carrying feature type.  Not saying that this PR must have this feature to pass, but if this PR doesn't address this, an issue should be made to make sure we get back to it at some point.

Reply:
This PR is only ScalaPB update. I think that the thing you're talking about is covered by `#164804993` (pivotal tracker issue)